### ISSUE TITLE
Create use q3vm as possible backend

### ISSUE THREAD

**Issue by mingodad** [Neat-Lang/neat#15]
I recently have been looking at https://github.com/jnz/q3vm and it seems to be a possible backend for neat and with it it can be interpreted for fast development/iteration.

**Comment by mingodad** [comment]
Just out of curiosity I replaced all `".+"[0]` and it does build fine see attached the diff.
[dad-char.diff.zip](https://github.com/Neat-Lang/neat/files/10267418/dad-char.diff.zip)

**Comment by FeepingCreature** [comment]
Yeah I added char support recently, they're all replaced on current master. It needs to build with the bootstrap compiler.

**Comment by FeepingCreature** [comment]
Having an interpreted backend is... complicated. I would like to be ABI compatible with compiled C, if possible, so I can easily call passed classes in macros. That said, unless it's *very* performant it wouldn't give any development time improvement, because the interpreted code would run slower than the compiler needs to build itself. It'd be a net loss, in other words.
